crates/rover-client/tests/client.rs

```diff
@@ -1,15 +1,27 @@
+use reqwest::blocking::Client;
 const STUDIO_PROD_API_ENDPOINT: &str = "https://graphql.api.apollographql.com/api/graphql";
 
+pub(crate) fn get_client() -> Client {
+    Client::builder()
+        .use_rustls_tls()
+        .tls_built_in_root_certs(true)
+        .gzip(true)
+        .brotli(true)
```

Comment:
Just for posterity, I wanted to make sure this uses the appropriate headers when determining whether or not it attempts the decompression, and indeed it does [according to the docs](https://docs.rs/reqwest/0.10.4/reqwest/struct.ClientBuilder.html#method.brotli).